.github/workflows/linux_meson.yml

```diff
@@ -248,3 +249,89 @@ jobs:
           export PYTHONOPTIMIZE=2
           python -m pytest --pyargs scipy -n2 --durations=10
           popd
+
+  #################################################################################
+  prerelease_deps_coverage_64bit_blas:
+    # TODO: re-enable ILP64 build.
+    name: Prerelease deps and 64-bit BLAS
+    # If using act to run CI locally the github object does not exist and the usual skipping should not be enforced
```

Comment:
Let's remove this line? We don't have to copy-paste that comment around, just explaining once near the top of the file is enough.